src/main/java/Deadline.java

```diff
@@ -0,0 +1,16 @@
+package main.java;
+
+public class Deadline extends Task {
```

Comment:
perhaps the deadline class does not need to be public? As deadline class is only accessed within the package. I think some of the other classes do not have to be public as well.